qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -432,21 +439,21 @@ class DefaultPoliciesTest extends BaseSpecification {
 
         "Image Vulnerabilities"           | 4.0f     | null |
                 // This makes sure it has at least 100 CVEs.
-                "Image \"quay.io/rhacs-eng/qa:struts-app\"" +
-                     " contains \\d{2,3}\\d+ CVEs with severities ranging between " +
```

Comment:
I think we should keep this regex (though it could be \\d{3,}). The CVE count changes too often to have it specific.

Reply:
changes done